src/sass/edu-benefits.scss

```diff
@@ -71,3 +71,26 @@ legend {
     margin-bottom: 0.5em;
   }
 }
+
+@media (max-width: 40.063em) {
+  .progress-box {
+    border:none;
+    padding-left: 2rem - .9375rem;
+    padding-right: 2rem - .9375rem;
+  }
+}
+
+.edu-benefits-mil-group.form-expanding-group-inner-enter, 
```

Comment:
These are adjustments to margins because setting something to 0 height apparently does not always keep margins inside that element from affecting elements outside of it. Without them you get some jumping of elements at the end of the transitions.